lib/modules/manager/bundler/parser.ts

```diff
@@ -0,0 +1,327 @@
+import Parser, { Query, SyntaxNode } from 'tree-sitter';
```

Comment:
it seems it's something like antlr?

does it make sense to use it for more languages?

Reply:
Yes, it is similar to Antlr, now it's gaining popularity among text editor enthusiasts (specifically, Neovim), and it powers GitHub's syntax highlight.